pkg/orchestration/wiring/ec2compute/v2/plugin.go

```diff
@@ -2,34 +2,59 @@ package v2
 
```

Comment:
Now we can get rid of v2 altogether and just call it ec2compute!

Reply:
just wait for V3 :)
just kidding - yeah, version is not needed anymore (but again, I would prefer this change to be a separate PR)

Reply:
In our glorious future of go modules that would be breaking the version number in the package path requirement. Also it's separate commits.